### PR TITLE
About concert page

### PR DESCRIPTION
**Pull request by Selevn**

### CODE REVIEW

File: src/js/components/Main/Concert/Concert.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from "prop-types"
+import {Container, Row, Col} from "react-bootstrap"
+
+import style from "./Concert.module.css"
+
+
+function Concert(props) {
+  return (
```
Comment (tanyamatesha) [inline src/js/components/Main/Concert/Concert.jsx]: please, rewrite without return.

File: src/js/components/Main/Concert/Concert.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      <div>
+        <Container className={style.mainContainer}>
+          <Row>
+            <Col sm={12} xs={12} md={7} lg={7} className={style.imageBlock}>
+              <img className={style.titleImage}
+                   src={props.concert.imgSrc}/>
```
Comment (tanyamatesha) [inline src/js/components/Main/Concert/Concert.jsx]: what do you think about destructuring props?

**Review by tanyamatesha — CHANGES_REQUESTED** [review]
Small fix. 
But what do you think about destructuring props? 
[For Example](https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0)

File: src/js/components/Main/Home/Home.jsx [hunk 3]
```diff
@@ -9,12 +11,17 @@ const Home = props => {
...
+              <input placeholder={languageSrc.search[langProps.language]} type="text" value={props.searchText}
+                     onChange={props.onInputChange}/>
+              <input type="button" onClick={() => {
+              }} value={languageSrc.search[langProps.language]}/>
+            </form>
             {/*подсказка при наборе*/}
             <div className="tip">
               <ul>
-                {props.concerts.map((item, index) => <li key={index}>{item.band}</li>)}
+                {props.concerts.map((item, index) => <li key={index}><Link to={"concert/" + item.id}>{item.band}</Link>
```
Comment (tanyamatesha) [inline src/js/components/Main/Home/Home.jsx]: it's not a very good idea to have index for keys. need to use `id`.

**Review by tanyamatesha — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 4]
```diff
@@ -1,3 +1,4 @@
 /node_modules
 /dist
-/.idea
\ No newline at end of file
+/.idea
+/src/img
```
Comment (wldyslw) [inline .gitignore:4]: Why aren't images supposed to be in repo?

**Review by wldyslw — COMMENTED** [review]
Don't forget to mark **changes requesting** comments as **resolved** after resolving them

File: .gitignore (hunk 4 above)
Comment (Selevn) [reply .gitignore:4]: It is template images, after DB connect they will adds in admin panel into db(or smth) and will be taken from it.
folder is empty
Moreover i still don't fix local images trouble (shame) and will return in need...

Comment (Selevn) [reply .gitignore:4]: Should i mark this comment as resolved?

**Review by tanyamatesha — APPROVED** [review]

**Pull request merged**